docs/core/compatibility/sdk/10.0/nu1510-pruned-references.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Breaking change - NU1510 is raised for direct references pruned by NuGet"
+description: "Learn about the breaking change in .NET 10 where NU1510 is raised for unnecessary direct package references."
+ms.date: 3/25/2025
+ai-usage: ai-assisted
+ms.custom: https://github.com/dotnet/docs/issues/45462
+---
+
+# NU1510 is raised for direct references pruned by NuGet
+
+Starting in .NET 10, NuGet raises a [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) when a project includes a direct package reference that overlaps with a framework-provided library and is not required.
+
+## Version introduced
+
+.NET 10 Preview 1
```

Comment:
Because this PR is new, is this the right preview?

Reply:
@BillWagner That's what was in the issue. 🤷🏻‍♂️